packages/app/src/cli/commands/app/dev.ts

```diff
@@ -63,6 +63,18 @@ export default class Dev extends Command {
       env: 'SHOPIFY_FLAG_NO_TUNNEL',
       default: false,
     }),
+    theme: Flags.string({
+      hidden: false,
+      char: 't',
+      description: 'Theme ID or name of the theme app extension host theme.',
+      env: 'SHOPIFY_FLAG_THEME',
+    }),
+    port: Flags.integer({
+      hidden: false,
+      char: 'p',
+      description: 'Local port of the theme app extension development server.',
+      env: 'SHOPIFY_FLAG_PORT',
+    }),
```

Comment:
We wanted to add a `port` property for future use when serving the extensions via localhost or custom tunnel 🤔 
Maybe we could have a more generic message here? or leave it like this and update it in the future? // @pepicrft

Reply:
@isaacroldan @amcaplan I believe both approaches work, we might use a more generic description or a different flag name.

Personally, as  developer will have different ports to manage, I believe we maybe could rename it to `--extension-port`. What do you think?

Reply:
Isn't there already a port for the UI extension dev console? And another port where UI extensions are actually served? This is a third kind of extension port. I think something like `--theme-app-extension-port`, while lengthy, might be best.

Reply:
Agree with @amcaplan. Also, couldn't the CLI just select a random port?

Reply:
I agree with @amcaplan as well, it's more flexible to keep the port attached to theme extensions, then we can keep the port flag available for future usage.

Regarding the usage of a random port, I'd prefer to keep this following the theme development server standard. Still, we might revisit it in a future PR applying the random port approach for both cases.